docs/users-guide.md

```diff
@@ -1,3 +1,78 @@
 # OpenShift Service Mesh Console User's Guide
 
-TODO
+The OpenShift Service Mesh Console (aka OSSMC) is an extension to the OpenShift Console which provides visibility into your Service Mesh. The functionality you see described here is very similar to that of the standalone Kiali Console (see https://kiali.io). In fact, you can still access the standalone Kiali Console if you wish. This User's Guide, however, will discuss the functionality that you see from within the OpenShift Console itself.
+
+:warning: The OSSMC [only supports a single tenant today](https://github.com/kiali/openshift-servicemesh-plugin/issues/28). Whether that tenant is configured to access only a subset of OpenShift projects or has access cluster-wide to all projects does not matter, however, only a single tenant can be accessed.
+
+# Overview
+
+The _Overview_ page provides a summary of your mesh by showing cards representing the namespaces participating in the mesh. Each namespace card has summary metric graphs and additional health details. There are links in the cards that take you to other pages within OSSMC.
```

Comment:
imo mentions of the tab name should be bold and not italicized.

Reply:
I don't know the standard doc way of doing it. Perhaps @JStickler can tell us.